src/Forms/ElementalAreaField.php

```diff
@@ -208,4 +212,55 @@ public function performReadonlyTransformation()
             ->setName($this->getName())
             ->addExtraClass('elemental-area--read-only');
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setSubmittedValue($value, $data = null)
+    {
+        // Content comes through as a JSON encoded list through a hidden field.
+        return $this->setValue(Convert::json2array($value));
```

Comment:
FYI we've agree to deprecate these JSON handling methods, from now on we may as well just use regular PHP json methods. RFC: https://github.com/silverstripe/silverstripe-framework/issues/8424